common/common.go

```diff
@@ -2,5 +2,6 @@ package common
 
 // Default Argo CD instance name
 const (
-	ArgoCDInstanceName = "openshift-gitops"
+	ArgoCDInstanceName           = "openshift-gitops"
+	EnvVar_disableDefaultInstall = "DISABLE_DEFAULT_ARGOCD_INSTANCE"
```

Comment:
If I am not mistaken, go naming conventions do not use underscores in consts/variable names.